package.json

```diff
@@ -76,23 +76,68 @@
           "command": "azure-iot-edge.buildSolution",
           "group": "edge@1"
         },
+        {
+          "when": "resourceFilename == deployment.template.debug.json",
+          "command": "azure-iot-edge.buildSolution",
+          "group": "edge@1"
+        },
         {
           "when": "resourceFilename == deployment.template.json",
           "command": "azure-iot-edge.buildAndPushSolution",
           "group": "edge@2"
         },
+        {
+          "when": "resourceFilename == deployment.template.debug.json",
+          "command": "azure-iot-edge.buildAndPushSolution",
+          "group": "edge@2"
+        },
         {
           "when": "resourceFilename == deployment.template.json",
           "command": "azure-iot-edge.buildAndRunSolution",
           "group": "edge@3"
         },
+        {
+          "when": "resourceFilename == deployment.template.debug.json",
+          "command": "azure-iot-edge.buildAndRunSolution",
+          "group": "edge@3"
+        },
         {
           "when": "resourceFilename == deployment.template.json",
           "command": "azure-iot-edge.generateDeployment",
           "group": "edge@4"
         },
         {
-          "when": "resourceFilename == deployment.json",
+          "when": "resourceFilename == deployment.template.debug.json",
```

Comment:
No more "Run Solution" context menu for deployment.json?

Reply:
Good catch. Will add back